scripts/qa_crowbarsetup.sh

```diff
@@ -357,6 +349,11 @@ function addsles12sp2testupdates
     # [[ $hacloud = 1 ]] && add_mount "SLE12-SP2-HA-Updates-test" \
     #     $distsuseip':/dist/ibs/SUSE:/Maintenance:/Test:/SLE-HA:/12-SP2:/x86_64/update/' \
     #     "$tftpboot_repos12sp2_dir/SLE12-SP2-HA-Updates-test/"
+    if isrepoworking SUSE-Enterprise-Storage-4-Updates-test ; then
+        [ -n "$deployceph" -a iscloudver 7plus ] && add_mount "SUSE-Enterprise-Storage-4-Updates-test" \
+            $distsuseip":/dist/ibs/SUSE:/Maintenance:/Test:/Storage:/4:/$arch/update/" \
```

Comment:
The log reports

````
+(qa_crowbarsetup.sh:353) addsles12sp2testupdates(): '[' -n '' -a iscloudver 7plus ']'
./scripts/qa_crowbarsetup.sh: line 353: [: too many arguments
````

Might want to be something like `iscloudver 7plus && [ -n "$deployceph" ] && add_mount ...` (untested)